sdk/core/core-client-rest/test/sendRequest.spec.ts

```diff
@@ -332,4 +332,23 @@ describe("sendRequest", () => {
       body: "test",
     });
   });
+
+  it("should have timeout option", async () => {
+    const mockPipeline: Pipeline = createEmptyPipeline();
+    mockPipeline.sendRequest = async (_client, request) => {
+      const contentType = request.headers.get("content-type");
+      assert.equal(contentType, "application/json");
+      const timeout = request.timeout;
+      assert.equal(timeout, 120);
+      return {
+        headers: createHttpHeaders(),
+      } as PipelineResponse;
+    };
+
+    await sendRequest("GET", mockBaseUrl, mockPipeline, {
+      contentType: "application/json",
+      timeout: 120,
+      body: "test",
+    });
+  });
```

Comment:
Also not sure if I should add unit test for the other options I added.

Reply:
I don't think we need to test the options per se, but maybe have one test for the `operationOptionsToRequestParameters` helper?